_posts/2017-09-01-tips-for-upgrading-rails-3-2-to-4.markdown

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title: "The definitive guide to upgrading from Rails 3.2 to 4.0"
```

Comment:
@mauro-oto Please titleize this title. `The Definitive Guide To Upgrading From Rails 3.2 To 4.0` or `The Definitive Guide to Upgrading from Rails 3.2 to 4.0`

Reply:
Done!